### ISSUE TITLE
[Android] View._onDetached called twice when app is suspended.

### ISSUE THREAD

**Issue by x4080** [NativeScript/NativeScript#1543]
Hi, I wondered if this is intended? To reproduce :

- just add listview with content
- I add console.log in listview-android
```
    public _onDetached(force?: boolean) {
  console.log('detached');
}
```

and when we click back button on android (to go to home screen), it will called twice. I dont know about iOS since going to home screen seems not doing suspending/ quitting

btw using back button in android seems still retain all the previous values in the javascript ? So the activity is not recreated (at least the javascript) ?

**Comment by hamorphis** [comment]
We are aware of this. It is a private method though. What is the scenario that you are trying to achieve?

**Comment by x4080** [comment]
I was creating custom component based on listview

**Comment by hamorphis** [comment]
OK. I will see whether I can fix this in a neat way. Stay tuned.

**Comment by x4080** [comment]
cool

**Comment by hamorphis** [comment]
@x4080 I tried what you described and _onDetached was called once. I also tried navigating forward from a page that contains a ListView and it was also called only once. I can't reproduce this. Can you send me a small sample app that does exhibit this behavior. Thanks.

**Comment by x4080** [comment]
What i did was exit the app with backspace. Sorry if im not describing it completely.

**Comment by x4080** [comment]
Hi, just wanted you guys to know that this problem still persists in 1.6 it will be fix in 1.7 ?
